integrationTests/testInitializer.go

```diff
@@ -0,0 +1,1931 @@
+package integrationTests
```

Comment:
unfortunately we can not reuse the elrond-go/integrationTests functionality as that functionality is glued on pointers not interfaces, so that is whywe need to redefine everything.

Reply:
we restructure those things which are pointers - and rewrite as interfaces.